Color.js

```diff
@@ -222,7 +222,9 @@ var Color = (function(window){
 	* @returns Color
 	*/
 	Color.prototype.clone = function(){
-		return new Color(this.decimal()).alpha(this.alpha());
+		let c = new Color(this.decimal());
```

Comment:
turn this to `var` (just because the rest of the module was written long before ecma 5) and i'll merge it

Reply:
done :)